WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackSetupRequiredViewModel.swift

```diff
@@ -0,0 +1,152 @@
+    }
+
+    func didTapRightBarButtonItem(in viewController: UIViewController?) {
+        guard let viewController = viewController else {
+            return
+        }
+        authentication.presentSupport(from: viewController, screen: .jetpackRequired)
+    }
+}
+
+extension JetpackSetupRequiredViewModel {
+    enum Localization {
+        static let title = NSLocalizedString("Connect Store", comment: "Title of the Jetpack setup required screen")
+        static let installJetpack = NSLocalizedString(
+            "Install Jetpack",
+            comment: "Button to install Jetpack from the Jetpack setup required screen"
+        )
+        static let connectJetpack = NSLocalizedString(
+            "Connect Jetpack",
+            comment: "Button to authorize Jetpack connection from the Jetpack setup required screen"
+        )
+        static let setupErrorMessage = NSLocalizedString(
+            "To use this app for %@ you'll need the free Jetpack plugin installed and connected on your store.",
+            comment: "Error message on the Jetpack setup required screen." +
+            "Reads like: To use this app for test.com you'll need...")
+        static let connectionErrorMessage = NSLocalizedString(
+            "To use this app for %@ you'll need to connect your store to Jetpack.",
+            comment: "Error message on the Jetpack setup required screen when Jetpack connection is missing." +
+            "Reads like: To use this app for test.com you'll need...")
+        static let setupSubtitle = NSLocalizedString(
+            "You’ll need your store credentials to begin the installation.",
+            comment: "Subtitle on the Jetpack setup required screen"
+        )
+        static let connectionSubtitle = NSLocalizedString(
+            "You’ll need your store credentials to begin the connection.",
+            comment: "Subtitle on the Jetpack setup required screen when only Jetpack connection is missing"
+        )
+        static let helpBarButtonItemTitle = NSLocalizedString("Help", comment: "Help button on Jetpack setup required screen.")
+        static let termsContent = NSLocalizedString(
+            "By tapping the Connect Jetpack button, you agree to our %1$@ and to %2$@ with WordPress.com.",
+            comment: "Content of the label at the end of the Jetpack setup required screen. " +
+            "Reads like: By tapping the Connect Jetpack button, you agree to our Terms of Service and to share details with WordPress.com.")
+        static let termsOfService = NSLocalizedString(
+            "Terms of Service",
+            comment: "The terms to be agreed upon when tapping the Connect Jetpack button on the Jetpack setup required screen."
+        )
+        static let shareDetails = NSLocalizedString(
+            "share details",
+            comment: "The action to be agreed upon when tapping the Connect Jetpack button on the Jetpack setup required screen."
+        )
+    }
+
+    enum Strings {
```

Comment:
Super nit: This consists of URLs, and maybe a name like `Links` or `Constants` will suit this better?